tests/keras_tests/feature_networks_tests/feature_networks/old_api_test.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import tensorflow as tf
+import numpy as np
+
+import model_compression_toolkit as mct
+from tests.keras_tests.feature_networks_tests.base_keras_feature_test import BaseKerasFeatureNetworkTest
+from tests.keras_tests.tpc_keras import get_weights_only_mp_tpc_keras
+from model_compression_toolkit.core.tpc_models.default_tpc.latest import get_op_quantization_configs
+
+
+keras = tf.keras
+layers = keras.layers
+
+
+class OldApiTest(BaseKerasFeatureNetworkTest):
+    def __init__(self, unit_test, mp_enable=False, gptq_enable=False):
+        super().__init__(unit_test, val_batch_size=1, num_calibration_iter=100)
```

Comment:
Why do we need 100 iterations?

Reply:
For better accuracy in the Compare method